pkg/clients/edgeconnect/client.go

```diff
@@ -213,10 +213,10 @@ func (c *client) GetEdgeConnect(edgeConnectId string) (GetResponse, error) {
 }
 
 // UpdateEdgeConnect updates existing edge connect hostPatterns and oauthClientId
-func (c *client) UpdateEdgeConnect(edgeConnectId, name string, hostPatterns []string, oauthClientId string) error {
+func (c *client) UpdateEdgeConnect(edgeConnectId, name string, hostPatterns []string, k8sAutomationHostPattern string, oauthClientId string) error {
```

Comment:
Here the same as the comment above from @0sewa0 

do we need to pass `k8sAutomationHostPattern` if it is always `edgeConnect.K8sAutomationHostPattern()`

Reply:
Wow I was lost, Sorry you are of course right haha

Reply:
A more general question is, would it make sense just to pass the whole `EdgeConnect` object and create a request for it?

I don't know, the only problem I see is that it might cause a import loop (now or in the future)